### ISSUE TITLE
Loss Function auswählen

### ISSUE THREAD

**Issue by AnnaDobelmann** [baalIsBack/Datenanalyse-und-Daten-Mining#7]
Aktuell nutzen wir nll_loss und crossEntropyLoss function in unseren Codes.
Es gilt herauszufinden (auszuprobieren), welche loss function besser funktioniert -> ggf. ist auch eine ganz andere Funktion besser.
